src/jmespath.net.parser/jmespath.net.parser.csproj

```diff
@@ -28,21 +28,21 @@
     </LexFile>
     <Compile Include="JmesPathScanner.g.cs" Condition="!Exists( 'JmesPathScanner.g.cs' )" />
     <Compile Include="JmesPathParser.g.cs" Condition="!Exists( 'JmesPathParser.g.cs' )" />
-	<Compile Include="GplexBuffers.cs" Condition="!Exists( 'GplexBuffers.cs' )" />
-	<Compile Include="ShiftReduceParserCode.cs" Condition="!Exists( 'ShiftReduceParserCode.cs' )" />
+    <Compile Include="GplexBuffers.cs" Condition="!Exists( 'GplexBuffers.cs' )" />
+    <Compile Include="ShiftReduceParserCode.cs" Condition="!Exists( 'ShiftReduceParserCode.cs' )" />
   </ItemGroup>
   <ItemGroup>
     <ProjectReference Include="..\jmespath.net.interop\jmespath.net.interop.csproj" />
   </ItemGroup>
 
   <ItemGroup Condition=" '$(TargetFramework)' == 'netstandard1.3' ">
-	  <!-- Use GplexBuffers and ShiftReduceParserCode from the net13 folder instead, which are modified for use of netstandard 1.3 -->
-	  <Compile Remove="ShiftReduceParserCode.cs" />
-	  <Compile Remove="GplexBuffers.cs" />
-	  <PackageReference Include="System.Reflection.TypeExtensions" Version="4.7.0" />
+      <!-- Use GplexBuffers and ShiftReduceParserCode from the net13 folder instead, which are modified for use of netstandard 1.3 -->

```

Comment:
Although reformatting is needed, please leave that out for another pull request, if any.